src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -222,80 +223,81 @@ export const commandTileIndexValidate = command({
+        files: tiffLocations.length,
+        outputs: outputTiles.size,
+      },
       'TileIndex: Manifest Assessed for Duplicates',
     );
 
     if (args.forceOutput || isArgo()) {
       await fsa.write('/tmp/tile-index-validate/input.geojson', {
         type: 'FeatureCollection',
-        features: locations.map((loc) => {
+        features: tiffLocations.map((loc) => {
           const epsg = args.sourceEpsg ?? loc.epsg;
           if (epsg == null) {
             logger.error({ source: loc.source }, 'TileIndex:Epsg:missing');
             return;
           }
           return Projection.get(epsg).boundsToGeoJsonFeature(Bounds.fromBbox(loc.bbox), {
             source: loc.source,
-            tileName: loc.tileName,
-            isDuplicate: (outputs.get(loc.tileName)?.length ?? 1) > 1,
+            tileName: loc.tileNames.join(', '),
           });
         }),
       });
       logger.info({ path: '/tmp/tile-index-validate/output.geojson' }, 'Write:InputGeoJson');
 
       await fsa.write('/tmp/tile-index-validate/output.geojson', {
         type: 'FeatureCollection',
-        features: [...outputs.values()].map((locs) => {
-          const firstLoc = locs[0];
-          if (firstLoc == null) throw new Error('Unable to extract tiff locations from: ' + args.location.join(', '));
-          const mapTileIndex = MapSheet.getMapTileIndex(firstLoc.tileName);
-          if (mapTileIndex == null) throw new Error('Failed to extract tile information from: ' + firstLoc.tileName);
+        features: [...outputTiles.keys()].map((key) => {
+          const mapTileIndex = MapSheet.getMapTileIndex(key);
+          if (mapTileIndex == null) throw new Error('Failed to extract tile information from: ' + key);
           return Projection.get(2193).boundsToGeoJsonFeature(Bounds.fromBbox(mapTileIndex.bbox), {
-            source: locs.map((l) => l.source),
-            tileName: firstLoc.tileName,
+            source: outputTiles.get(key)?.map((l) => l.source),
+            tileName: key,
           });
         }),
       });
       logger.info({ path: '/tmp/tile-index-validate/output.geojson' }, 'Write:OutputGeojson');
 
       await fsa.write(
         '/tmp/tile-index-validate/file-list.json',
-        [...outputs.values()].map((locs) => {
-          return { output: locs[0]?.tileName, input: locs.map((l) => l.source) };
+        [...outputTiles.keys()].map((key) => {
+          const locs = outputTiles.get(key);
+          return { output: key, input: locs?.map((l) => l.source) };
         }),
       );
-      logger.info({ path: '/tmp/tile-index-validate/file-list.json', count: outputs.size }, 'Write:FileList');
+      logger.info({ path: '/tmp/tile-index-validate/file-list.json', count: outputTiles.size }, 'Write:FileList');
     }
 
     let retileNeeded = false;
-    for (const val of outputs.values()) {
-      if (val.length < 2) continue;
+    for (const [tileName, tiffs] of outputTiles.entries()) {
+      if (tiffs.length < 2) continue;
```

Comment:
did wen want to make this `tiffs.length === 1` I think the tiffs.length === 0 is an assertion failure? so it should never happen.

Reply:
adds another level of nesting .. but OK

Reply:
another level of nesting?

```
if (tiffs.length === 0) throw new Error('output tile with no source tile: ...')
if (tiffs.length === 1) continue;

// ... mutilple input tiffs for a single output tiff
```

Reply:
my bad .. adding a level of nesting because I added a bug at the same time 👎🏿 
```
      if (tiffs.length === 1) {
        if (args.retile) {
```

Reply:
fixed